sources/web/datalab/static/dark.css

```diff
@@ -155,16 +155,25 @@ code {
 .google-visualization-table {
   color: #000;
 }
-.gchart-table-cell {
+.gchart-table-cell, tr.google-visualization-table-tr-odd {
```

Comment:
Can't this be combined with the style defined on line 166? They have the same body.

Regardless, I think that 'tr.google-visualization-table-tr-odd' should only occur in one of them.